src/query/sql/src/planner/binder/copy_into_location.rs

```diff
@@ -109,6 +109,15 @@ impl Binder {
                     .await
             }
         }?;
+        println!("qyery pln: {:?}", &query);
+        println!("qyery pln: {}", &query);
+        let mut is_ordered = false;
+        if let Plan::Query { s_expr, .. } = &query {
+            let p = s_expr.derive_relational_prop()?;
+            if !p.orderings.is_empty() || p.partition_orderings.is_some() {
```

Comment:
`if !p.orderings.is_empty() `

partition_orderings is not used for keeping order

Reply:
done